easybuild/easyblocks/a/abaqus.py

```diff
@@ -90,26 +91,46 @@ def install_step(self):
                 '___',
                 '\(\d+ MB\)',
             ]
-            std_qa = {
-                # disable installation of Tosca (6) and Isight (7)
-                "Isight\nEnter selection \(default: Next\):": '6\n7\n\n',
-                "(?<!Isight)\nEnter selection \(default: Next\):": '',
-                r"SIMULIA[0-9]*doc.*:": os.path.join(self.installdir, 'doc'),
-                r"SimulationServices.*:": os.path.join(self.installdir, 'sim'),
-                r"Choose the CODE installation directory.*:\n.*\n\n.*:": os.path.join(self.installdir, 'sim'),
-                r"SIMULIA/CAE.*:": os.path.join(self.installdir, 'cae'),
-                r"location of your Abaqus services \(solvers\).*(\n.*){8}:\s*": os.path.join(self.installdir, 'sim'),
-                r"Default.*SIMULIA/Commands\]:\s*": os.path.join(self.installdir, 'Commands'),
-                r"Default.*SIMULIA/CAE/plugins.*:\s*": os.path.join(self.installdir, 'plugins'),
-                r"Default.*SIMULIA/Isight.*:\s*": os.path.join(self.installdir, 'Isight'),
-                r"License Server [0-9]+\s*(\n.*){3}:": 'abaqusfea',  # bypass value for license server
-                r"License Server . \(redundant\)\s*(\n.*){3}:": '',
-                r"Please choose an action:": '1',
-                r"SIMULIA/Tosca.*:": os.path.join(self.installdir, 'tosca'),
-                r"location of your existing ANSA installation.*(\n.*){8}:": '',
-                r"FLUENT Path.*(\n.*){7}:": '',
-            }
-            run_cmd_qa('./StartTUI.sh', qa, no_qa=no_qa, std_qa=std_qa, log_all=True, simple=True, maxhits=100)
+            if LooseVersion(self.version) >= LooseVersion('2020'):
+                std_qa = {
+                    # disable installation of Isight (7) and enable installation of documentation (3)
+                    r"Isight\n\nEnter selection \(default: Next\):": '3\n7\n\n',
+                    # No Tosca (6 and 7), but do not trigger answering a later question that has no Tosca options
+                    r"21 \[\*\] Abaqus/CFD Solver\n\nEnter selection \(default: Next\):": '7\n8\n\n',
```

Comment:
@branfosj the comment says 6 and 7, but the input is 7 and 8 ?

Reply:
The comment is wrong - I'll correct it.

For reference, this is the list in question:
```
 -1 Select All
  0 Select None
  1 [*] Abaqus/Standard Solver
  2 [*] Abaqus/Explicit Solver
  3 [*] Cosimulation Services
  4 [*] Abaqus ODB API Services
  5 [*] Abaqus CAE
  6 [*] Abaqus Samples
  7 [*] Tosca Structure
  8 [*] Tosca Fluid
  9 [*] fe-safe
 10 [*] fe-safe interface to Abaqus 2019 ODBs
 11 [*] fe-safe interface to Abaqus 2018 ODBs
 12 [*] fe-safe interface to Abaqus 2017 ODBs
 13 [*] fe-safe interface to Abaqus 2016 ODBs
 14 [*] fe-safe materials and surface finish specifications
 15 [*] Material database server for fe-safe
 16 [*] fe-safe tutorial and sample data
 17 [*] fe-safe tutorial models for Abaqus
 18 [*] fe-safe tutorial models for I-DEAS
 19 [*] fe-safe tutorial models for Ansys
 20 [*] fe-safe tutorial models for NASTRAN
 21 [*] Abaqus/CFD Solver
```

Reply:
We should include the `7 .*Tosca Structure` and `8 .*Tosca Fluid` in the question pattern, to be robust against changing numbering going forward?

You can even grab then number from the question, like we do in the WRF easyblock

Reply:
Yes. I've changed it so that we pull the answer from the question for these blocks.